src/js/package.json

```diff
@@ -23,26 +24,29 @@
   "dependencies": {
     "@types/mocha": "7.0.2",
     "@types/node": "14.0.23",
-    "fast-crc32c": "2.0.0",
     "chokidar": "^3.4.3",
+    "fast-crc32c": "2.0.0",
```

Comment:
why are some lines reordered? Makes the diff more confusing to read

Reply:
this change is making by `npm install` when we install a new dependency, npm re-order the dependency in order to move shared dependencies to the top of the tree dependency, in this case when I installed the 'source-map-support' and 'nyc', this move the previous dependencies in order to improve the dependencies resolve.

https://medium.com/learnwithrahul/understanding-npm-dependency-resolution-84a24180901b